Tella/Domain/Entity/Report/SettingsModel.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+
+class SettingsModel: ObservableObject, Codable {
+    
+    @Published var offLineMode = false
+    @Published var quickDelete: Bool = false
+    @Published var deleteVault: Bool = false
+    @Published var deleteForms: Bool = false
+    @Published var deleteServerSettings: Bool = false
+    @Published var showRecentFiles: Bool = false
+    @Published var lockTimeout: LockTimeoutOption = .immediately
+    @Published var screenSecurity: Bool = true
+    
+    enum CodingKeys: CodingKey {
+        case offLineMode
+        case quickDelete
+        case deleteVault
+        case deleteForms
+        case deleteServerSettings
+        case showRecentFiles
+        case lockTimeout
+        case screenSecurity
+    }
+    
+    init() {
+        
```

Comment:
I think it's unuseful to add initialization within the initializer

If a property always takes the same initial value, provide a default value rather than setting a value within an initializer. 
https://docs.swift.org/swift-book/documentation/the-swift-programming-language/initialization/